Cargo.toml

```diff
@@ -19,7 +19,8 @@ members = [
     "ethers-signers",
     "ethers-core",
     "ethers-middleware",
-    "ethers-etherscan"
+    "ethers-etherscan",
+    "ethers-core/ethers-derive-eip712",
```

Comment:
Is this needed? I think we should just be re-exporting the macro/traits from ethers-core crate

Reply:
+1 on that, perhaps feature gated in core? default is enabled

Reply:
Added a feature gate under `ethers_core::types::transaction::mod`:

```rust
#[cfg(feature = "eip712")]
pub mod eip712;
```

Also included a quick comment in the docs for it, see c911087